addons/project_timesheet_holidays/tests/test_timesheet_holidays.py

```diff
@@ -53,6 +53,7 @@ def setUp(self):
         self.internal_project = self.env.company.internal_project_id
         self.internal_task_leaves = self.env.company.leave_timesheet_task_id
 
+        # timesheet_generate is true by default
```

Comment:
It's true by default, but the user is able to set it to False manually, now, right ? Then I think it would be nice to add a test with a leave type with the field set to False and ensure no timesheets are generated (unless a test already exists)

Reply:
Indeed, nvm. LGTM then :smile: